### PR TITLE
Archivelistwidgetitem

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

File: src/widgets/archivelistwidgetitem.cpp [hunk 1]
```diff
@@ -1,24 +1,16 @@
...
-    _ui.inspectButton->setToolTip(_ui.inspectButton->toolTip()
-                                   .arg(_ui.actionInspect->shortcut()
-                                        .toString(QKeySequence::NativeText)));
-    _ui.restoreButton->setToolTip(_ui.restoreButton->toolTip()
-                                   .arg(_ui.actionRestore->shortcut()
-                                        .toString(QKeySequence::NativeText)));
-    _ui.deleteButton->setToolTip(_ui.deleteButton->toolTip()
-                                   .arg(_ui.actionDelete->shortcut()
-                                        .toString(QKeySequence::NativeText)));
-
```
Comment (shinnok) [inline src/widgets/archivelistwidgetitem.cpp]: Why did you move these pieces around? The global style in the constructors should be:
1. UI initializations;
2. Setup/logic code;
3. Signals and slots;

Comment (gperciva) [inline src/widgets/archivelistwidgetitem.cpp]: Ok, I've changed it to use that style.
